### PR TITLE
feat: pinky swear seed phrase is backed up on snap warning modal

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

LLM-gen'd copy acting as a tacit agreement you as a user have saved your seed phrase and failure to do so means you will lose your fundus if you lose access to your MM installation.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

None

## Testing

- We are happy with the copy

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

## Screenshots (if applicable)

image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5260** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5260" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5260?utm_source=stack-comment).

**Comment by firebomb1** [comment]
Can these new strings/button captions be placed in `en/main.json` please? So we can translate them, they seem rather important.

File: src/components/Modals/Snaps/SnapConfirm.tsx [hunk 1]
```diff
@@ -62,6 +63,13 @@ export const SnapConfirm: React.FC<SnapConfirmProps> = ({ onClose }) => {
             The ShapeShift Multichain Snap is a new, experimental feature, provided "as is"
           </AlertDescription>
         </Alert>
+        <Alert status='error' borderRadius='lg' mb={4}>
+          <AlertIcon />
+          <AlertDescription>
+            Ensure you have backed up your MetaMask seed phrase. Any funds you send to non-EVM
+            chains using this Snap will be directed to the default account you set up with MetaMask.
```
Comment (woodenfurniture) [inline src/components/Modals/Snaps/SnapConfirm.tsx:70]: as @firebomb1 requested, lets put this in a translate and into main.json

**Review by woodenfurniture — APPROVED** [review]
lgtm, translate copy if you agree

File: src/components/Modals/Snaps/SnapConfirm.tsx (hunk 1 above)
Comment (firebomb1) [reply src/components/Modals/Snaps/SnapConfirm.tsx]: Pushed a commit to make that modal translatable, hope this helps. If not revert of course :D

Comment (gomesalexandre) [reply src/components/Modals/Snaps/SnapConfirm.tsx]: tyvm @firebomb1 ! Frontran me on this one, looking good 🐐

**Pull request merged**